### PR TITLE
Feature/include tag

### PR DESCRIPTION
**Pull request by JJmako**
PR related to https://github.com/nccgroup/ScoutSuite/issues/838

Added tags to IAM users.

### CODE REVIEW

File: ScoutSuite/providers/aws/facade/iam.py [hunk 1]
```diff
@@ -99,6 +99,7 @@ async def get_users(self):
         await get_and_set_concurrently(
             [functools.partial(self._get_and_set_inline_policies, iam_resource_type='user'),
              self._get_and_set_user_groups,
+             self._get_and_set_user_tags,
```
Comment (ramimac) [inline ScoutSuite/providers/aws/facade/iam.py:102]: Is this necessary? My understanding is that [list_users already contains the Tags](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html#IAM.Client.list_users), we shouldn't have to make all these additional calls?

File: ScoutSuite/providers/aws/resources/iam/users.py [hunk 2]
```diff
@@ -16,5 +16,6 @@ def _parse_user(self, raw_user):
         raw_user['id'] = raw_user.pop('UserId')
         raw_user['name'] = raw_user.pop('UserName')
         raw_user['arn'] = raw_user.pop('Arn')
-
+        if (len(raw_user['tags']['Tags']) > 0):
```
Comment (ramimac) [inline ScoutSuite/providers/aws/resources/iam/users.py:19]: If you don't need _get_and_set_user_tags, tags will be accessible directly in raw_user['Tags']

Comment (JJmako) [reply ScoutSuite/providers/aws/resources/iam/users.py:19]: I was triying to access ['Tags'] from HTML, but without this conversion was imposible for me

File: ScoutSuite/providers/aws/facade/iam.py (hunk 1 above)
Comment (JJmako) [reply ScoutSuite/providers/aws/facade/iam.py:102]: I had to call to "list_user_tags" for getting the tags. Without that call, the user don't contain the Tags.

Comment (lowSoA) [reply ScoutSuite/providers/aws/facade/iam.py:102]: I've run a quick test and can't see any tags being returned. I've looked into list_users with boto3 and also with the CLI (list-users) and same results, no tags, even though it's in the documentation as you've mentioned (https://docs.aws.amazon.com/cli/latest/reference/iam/list-users.html).

**Review by ramimac — APPROVED** [review]

**Comment by j4v** [comment]
I've modified the partial so it looks like the other Tags / resources.

**Pull request merged**